torch/_dynamo/decorators.py

```diff
@@ -160,6 +160,38 @@ def allow_in_graph(fn):
     return fn
 
 
+def mark_traceable(traceable_fn):
+    """
+    Like `allow_in_graph`, but with the following enhancements/differences:
```

Comment:
We should treat this docstring like the actual documentation we're going to give users. We're probably going to expose this directly to users (or via a torch.compiler.* API)

As so, we shouldn't mention allow_in_graph -- the first escape hatch API a user should interact with is probably mark_traceable. Users also don't need to know what the representation (uses flat_apply) looks like. We could probably toss it into a comment in the code though.

Reply:
Ah I actually intended this as summary for the developers (us), and was thinking to add a more comprehensive docstring once we put this into the public `torch.compiler` namespace, like what we have for `allow_in_graph` there. Maybe I can leave this as comment instead.
https://github.com/pytorch/pytorch/blob/1c26a95ae3fcb347ebdf36ef7719e6d819d0487f/torch/compiler/__init__.py#L53-L131

Reply:
Let's leave this as a comment instead of a docstring

Reply:
Updated.